tests/test_console_scripts.py

```diff
@@ -0,0 +1,56 @@
+"""tests of console_scripts entry_point (formerly scripts/gql-cli)"""
+import io
+import json
+
+import pytest
+
+from gql import __version__
+
+from .test_aiohttp import query1_server_answer, query1_server_answer_data, query1_str
+
+
+def test_cli_ep_version(script_runner):
+    ret = script_runner.run("gql-cli", "--version")
+
+    assert ret.success
+
+    assert ret.stdout == f"v{__version__}\n"
+    assert ret.stderr == ""
+
+
+@pytest.mark.asyncio
+@pytest.mark.script_launch_mode("subprocess")
+async def test_ep_aiohttp_using_cli(
```

Comment:
this is enough to get coverage back... but is perhaps overkill. i wasn't sure if there were any cli things that would exercise the full code code path without a running server.

Reply:
it's better do overkill than to underkill :smiley: , so that's fine